uitools/import/Esri/ArcGISRuntime/Toolkit/FloorFilter.qml

```diff
@@ -0,0 +1,599 @@
+                        visible: false
+                        source: "images/search.svg"
+                        width: height
+                        anchors {
+                            left: parent.left
+                            top: parent.top
+                            bottom: parent.bottom
+                            margins: 4
+                        }
+                    }
+                    ColorOverlay {
+                        anchors.fill: searchImg
+                        source: searchImg
+                        color: noResultsFoundLabel.color
+                    }
+                }
+
+                Label {
+                    id: noResultsFoundLabel
+                    objectName: "noResultsFoundLabel"
+                    text: "No results found"
+                    visible: false
+                    Layout.columnSpan: 3
+                    // expanding to the showAllFacilities cell (and its ToolSeparator) in case is not visible. This will not break the grid layout, otherwise there is a missing row.
+                    // no checks on listView visibility because they are opposite visibility, so one takes the other cell when it is visible.
+                    Layout.rowSpan: !showAllFacilities.visible ? 3 : 1
+                    Layout.fillWidth: true
+                    Layout.topMargin: 5
+                    Layout.bottomMargin: 5
+                    horizontalAlignment: Qt.AlignHCenter
+                    verticalAlignment: Qt.AlignVCenter
+                }
+
+                ListView {
+                    id: listView
+                    objectName: "listView"
+                    visible: true
+                    Layout.preferredHeight: 200
+                    Layout.columnSpan: 3
+                    cacheBuffer: Math.max(contentHeight,0)
+                    contentWidth: contentItem.childrenRect.width
+                    Layout.minimumWidth: contentItem.childrenRect.width
+                    Layout.fillWidth: true
+                    Layout.fillHeight: true
+                    Layout.minimumHeight: count ? contentHeight / count : 0
+                    Layout.maximumHeight: count ? contentHeight / count * 3 : 0
+                    Layout.topMargin: 5
+                    // span taking space of showAllFacilities and the related ToolSeparator
+                    Layout.rowSpan: showAllFacilities.visible ? 1 : 3
+                    ScrollBar.vertical: ScrollBar {}
+                    spacing: 5
+                    clip: true
+                    // scroll list view to selected item
+                    onCurrentIndexChanged: positionViewAtIndex(currentIndex, ListView.Visible)
+
+                    // delegate model used to implement the filtering functionality, by setting visible elements in the `filtered` group.
+                    model: DelegateModel {
+                        id: dm
+                        property Connections conn: Connections {
```

Comment:
Yes works as intended without an explicit property. 
For my understanding, why did I need to set into a property inside the `DelegateModel`?

Reply:
When left unassigned, it will be assigned to the "default" property: https://doc.qt.io/qt-5/qtqml-syntax-objectattributes.html#default-properties

For example, the default property on a QML `Item` is [children](https://doc.qt.io/qt-5/qml-qtquick-item.html#children-prop). I don't believe `DelegateModel` inherits from anything that allows is to have a default property.